### PR TITLE
gh #105 host code cleanup

### PR DESCRIPTION
**Pull request by ishthiyaqahmed**
code clean up for the host l1 test cases.

### CODE REVIEW

**Comment by drissya-k** [comment]
Use spaces instead of tabs for test_l1_dsHost_register.

**Review by drissya-k — APPROVED** [review]

File: src/test_l1_dsHost.c [hunk 1]
```diff
@@ -602,18 +583,18 @@ void test_l1_dsHost_positive_dsGetHostEDID(void) {
 void test_l1_dsHost_negative_dsGetHostEDID(void) {
     gTestID = 10;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
-	
+
     unsigned char edid[EDID_MAX_DATA_SIZE]= {0};  // Making an assumption about maximum EDID size.
     int length;
```
Comment (Anbukannadhasan) [inline src/test_l1_dsHost.c:588]: Initialize with default value "int length = 0;"

File: src/test_l1_dsHost.c [hunk 2]
```diff
@@ -374,7 +355,7 @@ void test_l1_dsHost_positive_dsGetCPUTemperature(void) {
 void test_l1_dsHost_negative_dsGetCPUTemperature(void) {
     gTestID = 6;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
-	
+
     float temperatureValue;
```
Comment (Anbukannadhasan) [inline src/test_l1_dsHost.c:359]: initialize with default value "float temperatureValue = 0.0f"

File: src/test_l1_dsHost.c [hunk 3]
```diff
@@ -323,12 +304,12 @@ void test_l1_dsHost_negative_dsHostTerm(void) {
 void test_l1_dsHost_positive_dsGetCPUTemperature(void) {
     gTestID = 5;
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
-	
+
     float temperatureValue;
```
Comment (Anbukannadhasan) [inline src/test_l1_dsHost.c:308]: initialize with default value "float temperatureValue = 0.0f"

**Review by Anbukannadhasan — COMMENTED** [review]
Added review comments

File: src/test_l1_dsHost.c (hunk 1 above)
Comment (ishthiyaqahmed) [reply src/test_l1_dsHost.c]: done

File: src/test_l1_dsHost.c (hunk 2 above)
Comment (ishthiyaqahmed) [reply src/test_l1_dsHost.c]: done

File: src/test_l1_dsHost.c (hunk 3 above)
Comment (ishthiyaqahmed) [reply src/test_l1_dsHost.c]: done

**Review by Anbukannadhasan — APPROVED** [review]
Approved

**Review by drissya-k — APPROVED** [review]

**Pull request merged**